src/App.js

```diff
@@ -1,53 +1,86 @@
 import React, { Component } from 'react';
-import Navbar from './components/Navbar'
-import MovieDetails from './components/MovieDetails'
-import MoviesContainer from './components/MoviesContainer'
+import Navbar from './components/Navbar';
+import MovieDetails from './components/MovieDetails';
+import MoviesContainer from './components/MoviesContainer';
 import './App.css';
 import movieDetails from './data/movieDetails';
-import moviesData from './data/movieData';
+import ErrorMessage from './components/ErrorMessage';
 
 class App extends Component{
   constructor() {
     super();
     this.state = {
       movieSelected: 0,
```

Comment:
It also looks like later in your code, this piece of state ends up being assigned `true` or `false`. I would recommend using a default value that is the same data type as what the assigned value would be later. So here, I would have the default value be `movieSelected: false`, not `0`. It won't change the functionality but it does make more sense for other devs coming into this code.